endpoint/endpointComp/Component.cdef

```diff
@@ -102,6 +102,8 @@ requires:
     {
         le_secStore.api
         modemServices/le_sim.api
+        modemServices/le_mdc.api
+        le_data.api                 [manual-start]
```

Comment:
What is the indentation and/or preferable style for this file?

Reply:
The Legato AF didn't have the formatting standards for definition files. Is it better to add four space between API and options?

Reply:
Let's use 4 spaces as the splitter.